src/Parser.js

```diff
@@ -0,0 +1,58 @@
+
+define(["ObjectsConfig"], function (ObjectsConfig) {
+    "use strict";
+
+    var Parser = function (xml) {
+        var _xml = xml;
+
+        this.getXml = function () {
+            return _xml;
+        }
+
+        var _xmlDom = (function (xml) {
```

Comment:
convert _xmlDom as property: this.configDom (and remove unnecessary getXmlDom method)
this.configDom = (new ObjectsConfig.DOMParser).parseFromString(xml, 'text/xml')